addons/stock_account/models/stock.py

```diff
@@ -717,3 +717,5 @@ class ProcurementGroup(models.Model):
     def _run_scheduler_tasks(self, use_new_cursor=False, company_id=False):
         super(ProcurementGroup, self)._run_scheduler_tasks(use_new_cursor=use_new_cursor, company_id=company_id)
         self.env['stock.move']._run_fifo_vacuum()
+        if use_new_cursor:
```

Comment:
This part is not strictly necessary since the cursor is closed just after:

https://github.com/odoo/odoo/blob/4a1a29c2ab697f7d3e2758cc496f4b8cb09ad618/addons/stock/models/procurement.py#L277

However, this is indeed probably safer to do it in case third-party modules would override the method.